modules/azure-core-cpp/1.14.1/overlay/.bazelrc

```diff
@@ -0,0 +1 @@
+common --@curl//:ssl_lib=openssl
```

Comment:
What's the plan for having users set this? Should this requirement be mentioned in a build file comment?

Reply:
The default CI setup has an anonymous module as the main module that depends only on your module. You need https://github.com/bazelbuild/bazel-central-registry/blob/main/docs/README.md#test-module to properly test with command line flags that reference other repos.